material-intro-screen/src/main/java/agency/tango/materialintroscreen/widgets/InkPageIndicator.java

```diff
@@ -130,7 +130,7 @@ public InkPageIndicator(Context context, AttributeSet attrs, int defStyle) {
         dotDiameter = typedArray
                 .getDimensionPixelSize(R.styleable.mis_InkPageIndicator_mis_dotDiameter,
                         DEFAULT_DOT_SIZE * density);
-        dotRadius = dotDiameter / 2;
+        dotRadius = dotDiameter / 2f;
```

Comment:
both dotRadius and haldDotRadius are float

I would put it to also to variable and use it instead of 2f